recipes/parmed/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "ParmEd" %}
+{% set version = "2.7.3" %}
+{% set md5 = "027cd3b738318c044164ff5ca06295e5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: ParmEd-{{ version }}.tar.gz
+  url: https://github.com/{{ name }}/{{ name }}/archive/{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+
+build:
+  number: 1
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - numpy
+    - setuptools
+
+  run:
```

Comment:
It's not entirely clear to me from looking at the repo what the requirements are.

Reply:
Hmmm ... same here. Would appear that only numpy is strictly required, but I'll run some tests to confirm.